### DISCUSSION TITLE
Anyway to get the RichEditBox's RTF to use unicode for non-English languages?

### DISCUSSION THREAD

**Discussion by AlexanderBlackman** [microsoft/microsoft-ui-xaml#10133] [Q&A]
When I type Chinese, things end up encoded in GB2312 not unicode.

Is there anyway to force unicode?  I know there's using\utf8 in the header instead of \ansi\ansicpg1252 works for manual rtf, but I don't know how to change it.

**Comment by gautambjain** [comment]
You may want to try the method shown here on a blank rich edit box. It is in C++ but it is achievable in C# too: https://stackoverflow.com/questions/1782409/unicode-rtf-text-in-richedit

**Reply by AlexanderBlackman** [reply]
Thank you, I'll give it a try tomorrow morning.
